.docker/Dockerfile

```diff
@@ -62,4 +63,8 @@ RUN ahab_dep=${AHAB_DEPENDENCIES} && \
 # Move the rest of the app over
 COPY . ${DARIA_DIR}
 
+# Build the js and css
+RUN yarn build
```

Comment:
the rub here is that these commands do a similar-to-webpack thing, where they watch for changes in development. but restoring the build definitely more valuable than getting this perfect, so ⏺️